Packs/Humio/Integrations/Humio/README.md

```diff
@@ -0,0 +1,811 @@
+Integration with Humio
+This integration was integrated and tested with version xx of Humio
+## Configure Humio on Demisto
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Humio.
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| url | Humio URL | True |
+| API-key | User API token | True |
+| insecure | Trust any certificate \(not secure\) | False |
+| proxy | Use system proxy settings | False |
+| queryParameter | Query to use to fetch incidents | False |
+| queryRepository | Fetch incidents from repository | False |
+| queryStartTime | Fetch incidents from | False |
+| queryTimeZoneOffsetMinutes | TimeZoneOffset in Minutes | False |
+| isFetch | Fetch incidents | False |
+| incidentType | Incident type | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Obtaining an API key
+Go to https://your-humio/settings and copy the API token. Example [https://cloud.humio.com/settings](https://cloud.humio.com/settings)
+
+## Fetch incidents
+The parameters used for fetch-incidents are only used if you want to use the fetch incidents feature. It is recommended to use alerts and notifiers in Humio to send this data to XSOAR via a webhook notifier instead. You can read more about the supported time-formats for backfilling [here](https://docs.humio.com/api/using-the-search-api-with-humio/#time-specification)
+
+## Commands
+You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### humio-query
+***
+Query the data from Humio
+
+
+#### Base Command
+
+`humio-query`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| repository | Repository to search | Required | 
+| queryString | Query string to use | Required | 
+| start | Relative or absolute (epoch) | Optional | 
+| end | Relative or absolute (epoch) | Optional | 
+| isLive | Answer with true, 1, t, y or yes | Optional | 
+| timeZoneOffsetMinutes | TimeZoneOffset in Minutes (default 0) | Optional | 
+| arguments | Additional arguments | Optional | 
```

Comment:
Lets give an example for additional arguments in README and in the description of the field in yml